R/download_dataset.R

```diff
@@ -0,0 +1,170 @@
+#' Download data for publication
+#'
+#' ...
+#'
+#' @param connection A valid connection to the ETN database.
+#' @param animal_project_code (string) Animal project to download data from.
+#' @param directory (string) Path to local download directory.
```

Comment:
I would change "Path" with "Relative path"

Reply:
And also mentioning the default value.

Reply:
This has now been updated + default is mentioned.